### PR TITLE
Added DriveTeleop cmd for drivetrain, changes to Robotbuilder, defaul…

### PR DESCRIPTION
**Pull request by cgjyang**
…t auton command

### CODE REVIEW

File: 2021/Robot2021/src/main/cpp/subsystems/Intake.cpp [hunk 1]
```diff
@@ -156,3 +157,26 @@ void Intake::SetDeployerSolenoid(bool extended)
 
     m_position.Set((extended) ? m_position.kForward : m_position.kReverse);
 }
+
+void Intake::MoveIntakeWithJoysticks(frc::XboxController *throttleJoystick)
```
Comment (cgjyang) [inline 2021/Robot2021/src/main/cpp/subsystems/Intake.cpp:161]: Variable and method name can be changed if needed

**Review by jmullins16 — COMMENTED** [review]
Looks pretty close. I think we want the intake speed to use the m_acquireSpeed (from the config file) in one direction and when the stick is pushed forward (y < 0), it should use the m_expelSpeed. As a minor item, the joysticks always come in the range [-1.0, 1.0], so checking to see if the y axis value is below 1 is redundant. And finally, using 0.05 as the threshold is going to make the sticks extremely sensitive--just resting thumbs on them will likely cause the intake to move. Giving a larger deadband (currently the region between 0.05 and +0.05) would be better. Something like 0.35 would divide the total Y deflection into 3 roughly equal regions.

**Comment by ashanawaz** [comment]
Looks good to me syntax wise.

**Review by ngupta-22 — COMMENTED** [review]
Looks good!

**Pull request merged**